src/builder/commands/ubuntu.rs

```diff
@@ -437,7 +437,7 @@ impl Distro {
             packages::Npm => Some(vec!()),
             // PHP
             packages::Php if self.has_php7() => {
-                Some(vec!("php-common", "php-cli"))
+                Some(vec!("php-common", "php-cli", "php-json", "php-zip"))
```

Comment:
Why do you need `php-json` and `php-zip`? Please add a comment into the source code.